doc/mgr/prometheus.rst

```diff
@@ -79,34 +79,46 @@ drive statistics, special series are output like this:
 
 ::
 
-    ceph_disk_occupation{ceph_daemon="osd.0",device="sdd",instance="myhost",job="ceph"}
+    ceph_disk_occupation{ceph_daemon="osd.0",device="sdd", exported_instance="myhost"}
```

Comment:
This says exported_instance but the latest code change is using the original plain "instance"

Reply:
This is what a user will see without setting honor_labels though. Maybe a note in the honor_labels section that this setting causes this label transformation?